public/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
Similar request - is it necessary to move these files under `public/` as part of this PR? If so no problem, if not it may be good to lump this change as part of another PR. I created https://github.com/seanforyou23/patternfly-seed/issues/10 for capturing these types of changes.